.gitignore

```diff
@@ -30,5 +30,4 @@ quicktests/overlaying/list_of_quicktests.json
 *.project
 
 # Visual Studio Code
-.settings/
-
+.vscode/
```

Comment:
vsc lets you know at the top if a new version comes out.  That said, I don't see the harm in keeping both.